cloudfront-dist/deployment/docker.template.env

```diff
@@ -23,6 +23,7 @@ ASSETS_URL=publicAssetBuketUrl
 METRICS_SERVICE=disabled
 STONLY_WID=disabled
 SUPPORT_WEB_ADDRESS=https://yourloops.zendesk.com/hc/requests/new
+CONTACT_SUPPORT_WEB_URL=https://yourloops.zendesk.com/hc/requests/new
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏I don't figure out why there is here a new env var which is the same as `SUPPORT_WEB_ADDRESS` ? Couldn't you reuse it ?